pkg/types/types.go

```diff
@@ -78,3 +81,8 @@ type TLSConfig struct {
 	Region      string `yaml:"region"`
 	AccessKeyID string `yaml:"access_key_id"`
 }
+
+type Gitlab struct {
+	GitLabInstance  string `yaml:"gitlab_instance"`
+	InstallationTag string `yaml:"installation_tag"`
```

Comment:
This should not be configurable, it's `openfaas-cloud` as a default.

Reply:
Ok, moving it to the default value. In the GitLab README this was explained as configurable allowing to use custom tag value. But makes sense to stick to the default option.